migrations/sql/V2019.09.05.1623__AddSessionSchema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE user_session
+(
+    session TEXT primary key,
+    user_id TEXT references users (id)
+);
+
+CREATE TABLE file(
+    hash TEXT primary key
+);
+
+CREATE TABLE session_files
+(
+    session   TEXT primary key,
+    file      TEXT,
+    type      TEXT
+);
+
+ALTER TABLE session_files ADD FOREIGN KEY (session) REFERENCES user_session (session);
```

Comment:
any reason this can't be done in the CREATE TABLE call like is done for user_session.user_id ?